service/http.directory/src/main/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceHandler.java

```diff
@@ -228,6 +244,31 @@ protected void doSessionOpened(HttpAcceptSession session) throws Exception {
         session.close(false);
     }
 
+    private boolean commonPath(String absolutePath, String string) {
+
+        String commonPath = "";
+        String[] paths = {absolutePath, string};
+        String[][] folders = new String[paths.length][];
+        for (int i = 0; i < paths.length; i++) {
+            folders[i] = paths[i].split("/");
```

Comment:
On windows machines the path is a different slash.  In fact it can change from OS to OS.  See file [separator](https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html)